canonical-websites/www.ubuntu.com
# 2655: Move /about/about-ubuntu/our-philosophy to /community/mission

# Note
**Please merge #2685 first**

## Done

- Moved /about/about-ubuntu/our-philosophy to /community/mission
- Updated copy according to [copydoc](https://docs.google.com/document/d/1KREEL0iRsg7G1Sol4J5v7EraNljDTiIYMem5_rQLtKE/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/community/mission
- Check that it matches the [copydoc](https://docs.google.com/document/d/1KREEL0iRsg7G1Sol4J5v7EraNljDTiIYMem5_rQLtKE/edit#)
- Check that /about/about-ubuntu/our-philosophy redirects to /community/mission

## Issue / Card

Fixes #2655
